coderd/notifications/manager.go

```diff
@@ -75,8 +76,7 @@ func WithTestClock(clock quartz.Clock) ManagerOption {
 //
 // helpers is a map of template helpers which are used to customize notification messages to use global settings like
 // access URL etc.
-func NewManager(cfg codersdk.NotificationsConfig, store Store, helpers template.FuncMap, metrics *Metrics, log slog.Logger, opts ...ManagerOption) (*Manager, error) {
-	// TODO(dannyk): add the ability to use multiple notification methods.
+func NewManager(cfg codersdk.NotificationsConfig, store Store, ps pubsub.Pubsub, helpers template.FuncMap, metrics *Metrics, log slog.Logger, opts ...ManagerOption) (*Manager, error) { // TODO(dannyk): add the ability to use multiple notification methods.
```

Comment:
@dannykopping is this TODO still valid? inbox is kinda another method.

Reply:
This is referring to sending to SMTP + webhook
But yeah this TODO can likely be deleted (it was more of a reminder for myself, but this topic is already on @stirby's radar).